contents/blog/startup-finance-without-finance.md

```diff
@@ -0,0 +1,210 @@
+- Ask for quarterly and annual reports to be generated, not just monthly. You'll need these for random things like investor data requests throughout the year. 
+- If you are a B2B SaaS company, ask your accountant to put your hosting costs in COGS ('cost of goods sold') not IT expenses. Otherwise it'll look like you have a ridiculous 99% profit margin or something. 
+- If you pay a year up front for an annual subscription, make sure your accountants 'amortize' this over the whole year. This means they'll put 1/12 of the annual cost into each month's expenses. They should also do this if a customer pays an annual contract up front - but you'll need to tell them this. 
+- You may hear 'accruals' vs. 'cash'-based accounting floating around - this basically means you either record the revenue when you issued the invoice vs. when you actually received the cash in the bank. We use accruals-based accounting, which is usually the sensible choice for B2B SaaS. 
+
+## Predicting the future - aka Financial Planning
+
+I lied when I said accounting was the good stuff - _this_ is the good stuff. Let's get our Wall Street on. 
+
+For this section, I'm going to assume you have your accounts set up in QuickBooks. Next, go to your account in Pry - the first thing you'll be prompted to do is connect Pry to your QuickBooks account, which should be straightforward. You can also connect your bank accounts directly if you want to - I like this because it means that Pry always has the latest bank account balances in one place, which is easier than logging into each one and adding up the total. 
+
+Once you've got your historical accounts showing in Pry (aka 'actuals'), it's time to build our financial plan! We're going to do three things:
+
+- A simple _revenue forecast_ - we'll use Google Sheets then connect this to Pry
+- A simple _expense forecast_ - we'll use Google Sheets then connect this to Pry
+- A _hiring plan_ - in Pry itself
+
+Before you start, [make a copy of this handy template](https://docs.google.com/spreadsheets/d/1ft0rTdvGHvSD4EOOl-bhNiQLtFdWdyLwrEnvLFq5Wxc/edit?usp=sharing) - this is an adapted version of the actual sheet we use at PostHog. 
+
+**Revenue forecast**
+
+Under the 'Revenue' tab, you'll see this is filled with dummy data. Feel free to adapt this as you need for your business - you might just need one line at this stage!
+
+(I prefer to model this stuff in Sheets, as Pry's 'Models' functionality is much slower and less intuitive.)
+
+Feel free to extend the model as far out as you like - at PostHog, we don't go further than 3 years out, and really it's only the next 12-24 months we really care about trying to get right.
+
+Finally, click Data -> Named Ranges and then select the cells where your total MRR is displayed (in the template, that's cells B19 to AE19). Call it something sensible like 'Total MRR'.  
+
+Once you're done, this is how to get it into Pry: 
```

Comment:
A screenshot would help break up this list, maybe at the end?

Reply:
I wasn't sure which way to go with images - either include several screenshots, or keep it screenshot-free was my thinking but maybe just one would look weird?

Reply:
One would definitely look weird, and integrating screenshots and lists difficult. Up to you, might be a way to break a few of the sections up if you felt it was too intense. I guess that is the life of a finance person 😅

Reply:
Curious what you think @andyvan-ph when you come to look at this next week - my inclination is to not use screenshots for this one because things tend to move around and I'm referencing relatively simple things, albeit in multiple products. But if that means the article is too dry and people won't want to read it then I'm very happy to add in some!

(I actually started adding in a few _hilarious_ memes before taking them out as well as they detracted)

Reply:
I think I'd so no to screenshots atm, but have a stronger view once we've fed through some changes and seen what it looks like on preview. Something that worked on the startup ops one was having a gif for each of the major sections. It fits with Charles tone of voice.